schemas/iso19139/src/main/plugin/iso19139/index-fields/language-default.xsl

```diff
@@ -776,7 +776,7 @@
                string="{*/@codeListValue}"
                store="true" index="true"/>
         <Field name="cl_{concat(local-name(), '_text')}"
-               string="{util:getCodelistTranslation(name(*), string(*/@codeListValue), string($isoLangId))}"
+               string="{util:getCodelistTranslation(name(.), string(*/@codeListValue), string($isoLangId))}"
```

Comment:
I was getting errors of multiple elements being passed to name() when it was using name(*).  I applied this fix however I don't believe it is the correct fix.  I attempted another fix where I changed */@codeListValue to @codeListValue in the for-each and other references which seems to make more sense to me however however it was generating lots of errors related to translating the values.  

Maybe someone understands why I was getting multiple elements being passed to name().

To test it I mostly just used the multilingual template supplied in the profile and replace the gmd:language